src/main/kotlin/de/itemis/mps/gradle/modelcheck/Plugin.kt

```diff
@@ -15,6 +15,7 @@ open class ModelCheckPluginExtensions: BasePluginExtensions() {
     var warningAsError = false
     var errorNoFail = false
     var junitFile: File? = null
+    var maxHeap: String? = null
```

Comment:
If I understood the syntax correctly, it means that heapsize is optional. Correct?

Reply:
Thats correct. All existing project will work without specifying it. If you do not specify it the JVM will try to figure out the max heap size by its usual means. Looking into `JAVA_OPS` env variable for instance or checking the OS memory available.